docs/Getting started/Authentication.md

```diff
@@ -12,13 +12,13 @@ You will require both of these in order to make requests to the API.
 
 ### I am a Whispir customer
 
-If you're a Whispir customer with access to Whispir Classic and Workflows by Whispir, you can obtain an API key through the Workflows platform by following the steps outlined [here](https://help.whispir.com/en/articles/4940972-api-keys-portal).
+If you're a Whispir customer with access to Whispir Classic and Workflows by Whispir, you can obtain an API key through the Workflows platform by following the steps outlined <!-- markdown-link-check-disable -->[here](https://help.whispir.com/en/articles/4940972-api-keys-portal)<!-- markdown-link-check-enable-->.
```

Comment:
Whispir requires sign-in to access certain help docs, this one is actually working.